SU2_CFD/include/solver_structure.hpp

```diff
@@ -11119,6 +11199,284 @@ class CWaveSolver : public CSolver {
   
 };
 
+/*! \class CHeatSolverFVM
+ *  \brief Main class for defining the finite-volume heat solver.
+ *  \author O. Burghardt
+ *  \version 5.0.0 "Raven"
+ *  \date January 19, 2018.
+ */
+class CHeatSolverFVM : public CSolver {
```

Comment:
Again, I am fine with this becoming the only heat solver (it is ok to remove old features that folks aren't using). Of course, it also sounds like we can also reuse some things between this and the new FVM Poisson solver written by @koodlyakshay (numerics routines come to mind, as one example). We can coordinate more on this.